modules/ROOT/pages/policy-mule3-add-remove-headers.adoc

```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: header, policy
 
-API Manager supports several policies for injecting or removing HTTP headers from a request or response. The policies take effect before sending the request or response.
+API Manager supports policies for injecting or removing HTTP headers from a request or response. The policies take effect before sending the request or response.
```

Comment:
"injecting or removing": I would think a more parallel phrase would be "inserting or removing" (or the pair "injecting" and "ejecting," although I think that sounds odd...).

Reply:
Further, if the subsequent title uses "add," then maybe we want "adding or removing."

Reply:
Yes, I went with adding. Thanks!